source/statements.tex

```diff
@@ -659,24 +659,24 @@
 exposition only; and
 
 \item
-\textit{begin-expr} and \textit{end-expr} are determined as follows:
+\placeholder{begin-expr} and \placeholder{end-expr} are determined as follows:
 
 \begin{itemize}
 \item if the \grammarterm{for-range-initializer} is an expression of
-array type \tcode{R}, \textit{begin-expr} and \textit{end-expr} are
+array type \tcode{R}, \placeholder{begin-expr} and \placeholder{end-expr} are
 \tcode{__range} and \tcode{__range + __bound}, respectively, where \tcode{__bound} is
-the array bound. If \tcode{R} is an array of unknown size or an array of
+the array bound. If \tcode{R} is an array of unknown bound or an array of
 incomplete type, the program is ill-formed;
 
 \item if the \grammarterm{for-range-initializer} is an expression of
 class type \tcode{C}, the \grammarterm{unqualified-id}{s}
 \tcode{begin} and \tcode{end} are looked up in the scope of \tcode{C}
 as if by class member access lookup~(\ref{basic.lookup.classref}), and if either
-(or both) finds at least one declaration, \grammarterm{begin-expr} and
-\grammarterm{end-expr} are \tcode{__range.begin()} and \tcode{__range.end()},
+(or both) finds at least one declaration, \placeholder{begin-expr} and
+\placeholder{end-expr} are \tcode{__range.begin()} and \tcode{__range.end()},
 respectively;
 
-\item otherwise, \textit{begin-expr} and \textit{end-expr} are \tcode{begin(__range)}
+\item otherwise, \placeholder{begin-expr} and \placeholder{end-expr} are \tcode{begin(__range)}
```

Comment:
`\xname` adds double underscores, so we don't have to spell those out literally, and it takes care to add some small amount of whitespace so the two characters don't run together. I'm not sure if that motivation is ultimately justified, it's just a thing we have.